libevdev.rb

```diff
@@ -1,15 +1,16 @@
 class Libevdev < Formula
   desc "Wrapper library for evdev devices"
   homepage "http://www.freedesktop.org"
+  url    "https://www.freedesktop.org/software/libevdev/libevdev-1.5.2.tar.xz"
+  sha256 "5ee2163656a61f5703cb5c08a05c9471ffb7b640bfbe2c55194ea50d908f629b"
+  # tag "linuxbrew"
+
   bottle do
     cellar :any_skip_relocation
+    revision 2
```

Comment:
Well, that was my first attempt to fix failure on Travis. But it looks like it is something else. The error message is rather weird:
```
1.00s==> brew install ./libevdev-1.5.2.x86_64_linux.bottle.tar.gz
Checking for Homebrew updates...
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
to set your account's default identity.
Omit --global to set the identity only in this repository.
fatal: empty ident name (for <linuxbrew@3abe360fba49.(none)>) not allowed
You need to set your committer info first
```